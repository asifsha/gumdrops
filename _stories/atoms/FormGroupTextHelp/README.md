The `<FormGroupTextHelp>` is used to give warning or hints on `<input>` elements. They must be wrapped by a `<FormGroup>` component.

The color of this component is dependent on the `FormGroup's context`.

**Example:**

```
<FormGroup>
    <FormGroupLabel  ... />
    <TextInput name="password" ... />
    <FormGroupTextHelp text="Your password is incorrect"/>
</FormGroup>
```