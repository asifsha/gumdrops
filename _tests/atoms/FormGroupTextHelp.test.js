/* globals mount */
import React from 'react';
import FormGroupTextHelp from '../../components/atoms/FormGroupTextHelp';

const defaultProps = {
    text: 'Oh no, errors!',
    className: 'much-error'
};

describe('Expect <FormGroupTextHelp>', () => {
    it('to render', () => {
        const wrapper = mount(<FormGroupTextHelp {...defaultProps} />);
        expect(wrapper).toMatchSnapshot();
    });
});
