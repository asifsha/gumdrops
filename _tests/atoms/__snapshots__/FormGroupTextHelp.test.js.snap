// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect <FormGroupTextHelp> to render 1`] = `
<FormGroupTextHelp
  className="much-error"
  text="Oh no, errors!"
>
  <small
    className="gds-form-group__text-help much-error"
  >
    Oh no, errors!
  </small>
</FormGroupTextHelp>
`;
